cpp/src/strings/slice.cu

```diff
@@ -326,7 +324,7 @@ std::unique_ptr<column> slice_strings(strings_column_view const& strings,
 {
   auto strings_count = strings.size();
   // If there aren't any rows, return an empty strings column
-  if (strings_count == 0) return empty_like(strings.parent());
+  if (strings_count == 0) { return make_empty_column(type_id::STRING); }
```

Comment:
I noticed `empty_like` is widely used to create empty columns. Is `make_empty_column` a new factory preferred over `empty_like`?

Reply:
I preferred to be explicit about the column type being returned since there was no reason for ambiguity here. Also, it allowed for removing an extra include.

Reply:
Thanks! I didn't even know `make_empty_column` exists. :smile: